tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -25,6 +26,11 @@
  */
 final class MethodArgumentSpaceFixerTest extends AbstractFixerTestCase
 {
+    /**
+     * @var MethodArgumentSpaceFixer
+     */
+    protected $fixer;
```

Comment:
do we need to redeclare this property, it already exists in the parent

Reply:
It's not very elegant but I'm not sure there's an easier way.